src/__tests__/events-compat.js

```diff
@@ -0,0 +1,27 @@
+import { h } from 'preact' // required by render
+import { fireEvent, render } from '..'
+import { forwardRef } from 'preact/compat' // required for this test to make sense
+
+test('calling `fireEvent` with `preact/compat` and onChange works too', () => {
+  const handler = jest.fn()
+
+  // forwardRef needs to be imported from preact/compat for this test to make sense.
+  // Preact behavior when using onChange is described here:
+  // https://preactjs.com/guide/v10/differences-to-react#use-oninput-instead-of-onchange
+  // We want to test if onChange event gets caught with fireEvent.change()
```

Comment:
```suggestion
  // Preact only matches React's aliasing of `onChange` when `preact/compat` is used
  // This test ensures this is supported properly with `fireEvent.change()`
```

is how I'd word this, as it doesn't have anything to do with `forwardRef` really.